### PR TITLE
Migrate logic to unpack htmlSerializer to asDict() method

### PR DESCRIPTION
**Pull request by nedn**
@progers 
@walter (I don't know his github account)

### CODE REVIEW

File: HTMLSerializer.js [hunk 1]
```diff
@@ -663,6 +663,33 @@ var HTMLSerializer = class {
...
+          'pseudoElementPlaceHolderIndex':
+              htmlSerializer.pseudoElementPlaceHolderIndex,
+          'pseudoElementTestingStyleIndex':
+              htmlSerializer.pseudoElementStyleTestingIndex,
+          'pseudoElementTestingStyleId':
+              htmlSerializer.pseudoElementTestingStyleId,
+          'unusedId': htmlSerializer.generateId(document),
+          'frameIndex': htmlSerializer.iframeFullyQualifiedName(window)
+    };
+export PATH=/Users/nednguyen/.npm-packages/bin/:$PATH
```
Comment (progers) [inline HTMLSerializer.js]: I think this is a mistake.

**Review by progers — COMMENTED** [review]
Looks great! Sorry for the delay, I've been out at Blinkon in Tokyo. If you can fix that one nit I'll approve and merge.

**Comment by shaper** [comment]
@nedn this is my github account fyi

**Comment by nedn** [comment]
@progers I removed that line, PTAL again.

**Pull request merged**